docs/reference/commandline/create.md

```diff
@@ -12,95 +12,109 @@ parent = "smn_cli"
                                     'container:<name|id>': reuse another container's network stack
                                     'host': use the Docker host network stack
                                     '<network-name>|<network-id>': connect to a user-defined network
-      --net-alias=[]                Add network-scoped alias for the container
-      --oom-kill-disable            Whether to disable OOM Killer for the container or not
-      --oom-score-adj=0             Tune the host's OOM preferences for containers (accepts -1000 to 1000)
-      -P, --publish-all             Publish all exposed ports to random ports
-      -p, --publish=[]              Publish a container's port(s) to the host
-      --pid=""                      PID namespace to use
-      --pids-limit=-1                Tune container pids limit (set -1 for unlimited), kernel >= 4.3
+      --net-alias value             Add network-scoped alias for the container (default [])
+      --no-healthcheck              Disable any container-specified HEALTHCHECK
+      --oom-kill-disable            Disable OOM Killer
+      --oom-score-adj int           Tune host's OOM preferences (-1000 to 1000)
+      --pid string                  PID namespace to use
+      --pids-limit int              Tune container pids limit (set -1 for unlimited)
       --privileged                  Give extended privileges to this container
+  -p, --publish value               Publish a container's port(s) to the host (default [])
+  -P, --publish-all                 Publish all exposed ports to random ports
       --read-only                   Mount the container's root filesystem as read only
-      --restart="no"                Restart policy (no, on-failure[:max-retry], always, unless-stopped)
-      --runtime=""                  Name of the runtime to be used for that container
-      --security-opt=[]             Security options
-      --stop-signal="SIGTERM"       Signal to stop a container
-      --shm-size=[]                 Size of `/dev/shm`. The format is `<number><unit>`. `number` must be greater than `0`.  Unit is optional and can be `b` (bytes), `k` (kilobytes), `m` (megabytes), or `g` (gigabytes). If you omit the unit, the system uses bytes. If you omit the size entirely, the system uses `64m`.
-      --storage-opt=[]              Set storage driver options per container
-      --sysctl[=*[]*]]              Configure namespaced kernel parameters at runtime
-      -t, --tty                     Allocate a pseudo-TTY
-      -u, --user=""                 Username or UID
-      --userns=""                   Container user namespace
+      --restart string              Restart policy to apply when a container exits (default "no")
+      --runtime string              Runtime to use for this container
+      --security-opt value          Security Options (default [])
+      --shm-size string             Size of /dev/shm, default value is 64MB.
```

Comment:
Hm, wasnt there a change for the size recently, or was that for tmpfs? Need to check